### PR TITLE
[ALKES] change url api integration from poslog

### PR DESCRIPTION
**Pull request by imamdev93**
### Overview
- change  url api integration from `https://jabarajeg.poslogistics.co.id` to `http://stori.poslogistics.co.id/api/alkes`

### Link Issue
- https://app.clickup.com/t/35rk8ar

### Todo
- change environment `WMS_JABAR_BASE_URL` and `WMS_JABAR_API_KEY`

### Evidence
-title: [ALKES] change url api integration from poslog
-project: Pikobar Logistik
-participants: @imamdev93 @ayocodingit @rachadiannovansyah

### CODE REVIEW

File: app/Usage.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ static function getLogisticStock($param, $api, $baseApi)
     {
         $apiKey = PoslogProduct::isDashboardAPI($baseApi) ? config('dashboardexecutive.key') : config('wmsjabar.key');
         $apiLink = PoslogProduct::isDashboardAPI($baseApi) ? config('dashboardexecutive.url') : config('wmsjabar.url');
-        $apiFunction = $api ? $api : '/api/soh_fmaterialgroup';
+        $apiFunction = $api ? $api : '/index.php?route=soh_fmaterialgroup';
```
Comment (ayocodingit) [inline app/Usage.php:38]: mau nanya aja mang `/index.php?route=soh_fmaterialgroup` itu apa yah, sama aman gak yah .php nya terlihat

Comment (imamdev93) [reply app/Usage.php:38]: itu url dari poslog nya mang. `{{url}}/index.php`

Comment (ayocodingit) [reply app/Usage.php:38]: kalau url di config nya env nya misalnya gini gimana yah https://blabla/index.php gimana mang?

Comment (imamdev93) [reply app/Usage.php]: bisa mang, disini ya [d3cc1f0](https://github.com/jabardigitalservice/pikobar-logistik-api/pull/462/commits/d3cc1f005a7a925beed1e26ea4fa260f8b095ca2)

**Review by ayocodingit — APPROVED** [review]

**Review by rachadiannovansyah — APPROVED** [review]

**Pull request merged**